app/controllers/admin/users/payouts_controller.rb

```diff
@@ -81,13 +81,31 @@ def sync_all
   end
 
   def pause
-    @user.update!(payouts_paused_internally: true)
+    reason = params[:reason].presence || "Manual pause by admin"
```

Comment:
The corresponding front-end changes seem to be missing here? i.e. changes for an admin to enter the `reason` when pausing payouts?